### PR TITLE
Add cmd/crd-schema-gen

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: cmd/crd-schema-gen/main.go [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	apisDir := flag.String("apis-dir", "pkg/apis", "the Golang API directory")
```
Comment (mfojtik) [inline cmd/crd-schema-gen/main.go]: please clarify the description more

File: cmd/crd-schema-gen/main.go [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+	apis := flag.String("apis", "*", "the apis to generate from the apis-dir, in bash glob syntax")
+	manifestDir := flag.String("manifest-dir", "manifests", "the directory with existing CRD manifests")
```
Comment (mfojtik) [inline cmd/crd-schema-gen/main.go]: nit: `manifests`

File: cmd/crd-schema-gen/main.go [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+	outputDir := flag.String("output-dir", "", "optional directory to output the kubebuilder CRDs. By default a temporary directory is used.")
+	verifyOnly := flag.Bool("verify-only", false, "do not write files, only compare and return with return code 1 if dirty")
```
Comment (mfojtik) [inline cmd/crd-schema-gen/main.go]: sound like `dry-run` ?

File: cmd/crd-schema-gen/main.go (hunk 1 above)
Comment (sttts) [reply cmd/crd-schema-gen/main.go]: done

File: cmd/crd-schema-gen/main.go (hunk 3 above)
Comment (sttts) [reply cmd/crd-schema-gen/main.go]: called verify-only elsewhere. It's more than dry-run, fails on differences.

**Pull request merged**